### ISSUE TITLE
No Data in Grafana

### ISSUE THREAD

**Issue by wilkadam** [friendlyFriend4000/prometheus-immich-exporter#20]
Currently, after importing the dashboard, the "No data" is displayed. My prometheus is configured correctly, I can see that data is coming to it, but dashboard seems to not work. I am using Immich version 1.119.0.

![image](https://github.com/user-attachments/assets/bfc9ea0b-0a48-4884-98ab-d92811856f10)

**Comment by friendlyFriend4000** [comment]
I have just updated to immich to 1.190.1 and it seems to work on my machine. Let me know if you are strill struggling to get data.

**Comment by wilkadam** [comment]
Yes, I am strill struggling. Are you sure about the version? The newest is 119.1, I am using 119.0. The question is if I am using my own prometheus, not your prometheus-immich-exporter image, is it possible to even get the data? I wanted to just use your dashboards from grafana directory, but I can still see "no data"
